### PR TITLE
feat(lint): update eslint to v6.4.0

### PR DESCRIPTION
**Pull request by belgac**
this updates eslint and the typescript eslint parser

### CODE REVIEW

File: packages/build-tools/webpack-config/plugins/config/pwaManifest.js [hunk 1]
```diff
@@ -7,42 +7,40 @@ const { target, manifest: { name, shortName } = {} } = require('../../webpackPkg
...
-        destination: path.join('static', 'icons', 'android'),
-      },
-    ],
-    theme_color: '#76beea',
-    background_color: '#76beea',
-  },
-  pkgManifest,
-));
+module.exports = target === 'web' && new WebpackPwaManifest({
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/plugins/config/pwaManifest.js]: remove blank line

File: packages/build-tools/webpack-config/plugins/config/webpackDefinePlugin.js [hunk 2]
```diff
@@ -3,18 +3,16 @@ const { isEmpty } = require('ramda');
...
 
 module.exports = target === 'web' && new webpack.DefinePlugin({
-  'process.env': Object.assign(
-    {},
-    {
-      NODE_ENV: JSON.stringify(process.env.NODE_ENV),
-    },
-    defineEnv && !isEmpty(defineEnv)
+  'process.env': {
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/plugins/config/webpackDefinePlugin.js]: remove blank line

File: packages/build-tools/webpack-config/plugins/config/workBoxPlugin.js [hunk 3]
```diff
@@ -25,11 +25,9 @@ const dontCacheBustUrlsMatching = {
...
-module.exports = target === 'web' && process.env.NODE_ENV === 'production' && !(isEmpty(workboxConfig) || isNil(workboxConfig)) && new WorkboxPlugin.InjectManifest(Object.assign(
-  {},
-  workboxConfig,
-  {
-    include,
-  },
-  dontCacheBustUrlsMatching,
-));
+module.exports = target === 'web' && process.env.NODE_ENV === 'production' && !(isEmpty(workboxConfig) || isNil(workboxConfig)) && new WorkboxPlugin.InjectManifest({
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/plugins/config/workBoxPlugin.js]: remove blank line

File: packages/ge-scripts/inject-eslint-config/index.js [hunk 4]
```diff
@@ -2,18 +2,16 @@ const path = require('path');
...
-    pkg.eslintConfig || {},
-    {
-      extends: [
-        getPath('@goldwasserexchange/eslint-config'),
-      ].filter(Boolean),
-    }
-  ),
+const addEsLintConfig = (pkg) => ({
+  eslintConfig: {
+
```
Comment (jogold) [inline packages/ge-scripts/inject-eslint-config/index.js]: remove blank line

File: packages/build-tools/babel-preset/transformImports/index.js [hunk 5]
```diff
@@ -22,9 +22,9 @@ module.exports = () => {
   const babelTarget = getBabelTarget();
   return [
     require.resolve('babel-plugin-transform-imports'),
-    Object.assign(
-      {},
-      goldwasserExchangeUtilsTransform(babelTarget),
-    ),
+    {
+
```
Comment (jogold) [inline packages/build-tools/babel-preset/transformImports/index.js]: remove blank line

File: packages/build-tools/webpack-config/module/cssRules/cssLoader.js [hunk 6]
```diff
@@ -1,21 +1,19 @@
...
   loader: require.resolve('css-loader'),
-  options: Object.assign(
-    {},
-    {
-      importLoaders: 1,
-      modules: true,
-    },
-    process.env.NODE_ENV === 'development'
+  options: {
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/module/cssRules/cssLoader.js]: remove blank line

File: packages/build-tools/webpack-config/optimization.js [hunk 7]
```diff
@@ -2,9 +2,9 @@ const TerserPlugin = require('terser-webpack-plugin');
 const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const { target } = require('./webpackPkgConfig');
 
-module.exports = Object.assign(
-  {},
-  process.env.NODE_ENV === 'production'
+module.exports = {
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/optimization.js]: remove blank line

File: packages/build-tools/webpack-config/plugins/config/htmlWebpackPlugin.js [hunk 8]
```diff
@@ -2,14 +2,12 @@ const HtmlWebpackPlugin = require('html-webpack-plugin');
...
-module.exports = target === 'web' && new HtmlWebpackPlugin(Object.assign(
-  {},
-  {
-    inject: true,
-    env: process.env,
-    template: htmlTemplate || `${fromRoot(getPkgSrcDir())}/index.ejs`,
-  },
-  process.env.NODE_ENV === 'production'
+module.exports = target === 'web' && new HtmlWebpackPlugin({
+
```
Comment (jogold) [inline packages/build-tools/webpack-config/plugins/config/htmlWebpackPlugin.js]: remove blank line

**Comment by belgac** [comment]
ok good to know eslint fix can add blank lines :/

**Comment by jogold** [comment]
Updated PR title, please add a `BREAKING CHANGE` when squashing.

**Pull request merged**